package.json

```diff
@@ -4,6 +4,7 @@
   "description": "GraphQL Backend for Human Connection",
   "main": "src/index.js",
   "scripts": {
+    "eslint": "eslint src --config .eslintrc.js",
```

Comment:
I would also lint on precommit

Reply:
I would not. ["It’ll block the developer from committing the code if it doesn’t pass the rules"](https://medium.com/@shettyrahul8june/how-to-run-eslint-using-pre-commit-hook-25984fbce17e) I want **less** obstacles for people to commit and push their code instead of additional obstacles.

Reply:
Then we should consider using prettier to Auto fix Code on Save.